src/main/webapp/assets/modules/post-contest/postContestRewardDirective.js

```diff
@@ -0,0 +1,10 @@
+techlooper.directive("postContestReward", function ($http, utils) {
```

Comment:
'techlooper' is not defined.<br>'utils' is defined but never used.<br>'$http' is defined but never used.